docs/cel.md

```diff
@@ -29,20 +29,21 @@ be represented in CEL, requiring the usage of custom CEL functions. Below is a
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Variables**   |                                                                                                                                                                                                                           |                                                                                                                                                                   |
+| `now`           | `google.protobuf.Timestamp`                                                                                                                                                                                               | The current timestamp, computed once per expression (i.e., `now == now`).                                                                                         |
+| `this`          | `dyn`                                                                                                                                                                                                                     | The current value, referring to either a message or field's value.                                                                                                |
+| **Functions**   |                                                                                                                                                                                                                           |                                                                                                                                                                   |
+| `isEmail`       | `string.isEmail() -> bool`                                                                                                                                                                                                | Test whether the string is a valid email address.                                                                                                                 |
+| `isHostAndPort` | `string.isHostAndPort(bool) -> bool`                                                                                                                                                                                      | Test whether the string is a valid host/port pair. If the argument is true, the port is required, otherwise if false the port suffix is optional.                 |
```

Comment:
```suggestion
| `isHostAndPort` | `string.isHostAndPort(bool) -> bool`                                                                                                                                                                                      | Test whether the string is a valid host/port pair. If the argument is true (default), the port is required, otherwise if false the port suffix is optional.                 |
```

Reply:
Didn't know if we wanted to support a no-arg caller which defaults to requiring the port.

Reply:
It's technically not a default for the function (just the value used in the standard constraint).